README.md

```diff
@@ -257,6 +257,7 @@ The SDK has built-in support for handling Biometric encryption. To set the defau
 > Notes:
 > - The SDK does not check which biometrics are enrolled on the user's device. Please check this using https://developer.android.com/reference/android/hardware/biometrics/BiometricManager#canAuthenticate(int) before setting the appropriate security level
 > - The SDK automatically deletes Token entries stored using invalidated biometric keys.
+> - Biometric Credentials should only be fetched using async APIs in `Credential` class.
```

Comment:
Does the sync version throw an exception when biometrics are required? If so, you might want to indicate that, or reference the specific exception that is thrown, or maybe reference the API docs where this detail can be included.

Reply:
It does. I will make this change to specify the exception thrown.